misk/src/main/kotlin/misk/web/proxy/WebProxyEntry.kt

```diff
@@ -38,5 +38,6 @@ fun WebProxyEntry(
   url_path_prefix: String,
   web_proxy_url: String
 ) : WebProxyEntry {
+//  TODO(adrw) update all HTTPUrl.parse -> get (no bang bang required) https://github.com/square/misk/issues/419
```

Comment:
I’ll show you tools so it’s easier to just make fixes like this vs. writing comments

TODOs should be hard!

Reply:
Will submit another PR addressing this #419